src/styles.scss

```diff
@@ -0,0 +1,23 @@
+@import "./styles/utils/_variables.scss";
+@import "./styles/utils/_reset_links.scss";
+@import "./styles/blocks/card.scss";
+@import "./styles/blocks/catalog.scss";
+@import "./styles/blocks/header.scss";
+@import "./styles/blocks/nav.scss";
+@import "./styles/blocks/reset.scss";
+@import "./styles/blocks/stars.scss";
+
+.is-active {
+  color: #00acdc;
+
```

Comment:
you have this color like a variable in utils/_variables.scc : $color-blue: #00acdc;